src/applications/gi-sandbox/components/profile/BenefitsForm.jsx

```diff
@@ -140,7 +130,7 @@ export class BenefitsForm extends React.Component {
           />
           <div>
             {this.props.militaryStatus === 'active duty' &&
-              chapter33Check && (
+              this.props.giBillChapter === '33' && (
                 <div className="military-status-info warning form-group">
                   <i className="fa fa-warning" />
```

Comment:
```suggestion
                  <i className="fa fa-warning" aria-hidden="true" />
```

Reply:
While I do agree with this, I don't want to make any actual changes in this PR other than swapping the directories.

Reply:
Is this going into production?